### PR TITLE
Fix quality issues

### PR DESCRIPTION
**Pull request by tehreem-sadat**
**Ticket Link:**
https://edlyio.atlassian.net/browse/EDE-715

### CODE REVIEW

File: ecommerce/courses/models.py [hunk 1]
```diff
@@ -121,7 +121,7 @@ def enrollment_code_product(self):
                 return enrollment_code
         return None
 
-    def get_course_seat_name(self, certificate_type, id_verification_required):
+    def get_course_seat_name(self):
```
Comment (danialmalik) [inline ecommerce/courses/models.py:124]: Just make sure that this model isn't being used to inherit other classes.

File: ecommerce/extensions/payment/tests/processors/test_authorizenet.py [hunk 2]
```diff
@@ -65,19 +63,21 @@ def _get_refund_error_response_data(self, response_template, reference_transacti
...
         """
-        token_api_response = objectify.fromstring(hosted_payment_token_response_template)
-        mock_controller.return_value.getresponse.return_value = token_api_response
-        actual_data = self.processor.get_transaction_parameters(self.basket, request=self.request)
-        expected_data = {
-            'payment_page_url': self.processor.authorizenet_redirect_url,
-            'token': "test_token"
-        }
-        self.assertDictEqual(actual_data, expected_data)
+        with patch(
```
Comment (danialmalik) [inline ecommerce/extensions/payment/tests/processors/test_authorizenet.py:70]: Why did we need to change the `patch` from being a `decorator` to a `with` call?

File: ecommerce/extensions/payment/tests/views/test_authorizenet.py [hunk 3]
```diff
@@ -122,7 +123,10 @@ def test_notification_without_transaction_id(self, mock_logger):
             'Recieved AuthorizeNet transaction notification without transaction_id')
 
     @patch('ecommerce.extensions.payment.views.authorizenet.logger', autospec=True)
-    @patch('ecommerce.extensions.payment.views.authorizenet.AuthorizeNetNotificationView.payment_processor', autospec=True)
+    @patch(
+        'ecommerce.extensions.payment.views.authorizenet.AuthorizeNetNotificationView.payment_processor',
+        autospec=True
+    )
```
Comment (danialmalik) [inline ecommerce/extensions/payment/tests/views/test_authorizenet.py:129]: If changing the `patch` decorator call into a `with` call was to resolve the line length issues, you can resolve that same way you have done here.

File: ecommerce/courses/models.py (hunk 1 above)
Comment (tehreem-sadat) [reply ecommerce/courses/models.py:124]: I have fixed it everywhere.

File: ecommerce/extensions/payment/tests/processors/test_authorizenet.py (hunk 2 above)
Comment (tehreem-sadat) [reply ecommerce/extensions/payment/tests/processors/test_authorizenet.py:70]: To resolve [Arguments number differs from base method](http://pylint-messages.wikidot.com/messages:w0221) error as here we are overriding base-class **PaymentProcessorTestCaseMixin** method.

File: ecommerce/extensions/payment/tests/views/test_authorizenet.py (hunk 3 above)
Comment (tehreem-sadat) [reply ecommerce/extensions/payment/tests/views/test_authorizenet.py:129]: replied in previous comment.

**Pull request merged**